operations/cloudformation-templates/mozdef_user/mozdef_user.json

```diff
@@ -170,6 +172,7 @@
           "https://github.com/mozilla/mozdef-gen2-private/blob/f6557358d7a224197225538a5d9af0a59fb343b5/mq/esworker.infosecsqs.conf#L6",
           "https://github.com/mozilla/mozdef-gen2-private/blob/f6557358d7a224197225538a5d9af0a59fb343b5/mq/esworker.nubis.conf#L6",
           "https://github.com/mozilla/mozdef-gen2-private/blob/f6557358d7a224197225538a5d9af0a59fb343b5/mq/esworker.sqs.py#L570",
+          "https://github.com/mozilla/mig-deploy/blob/81ba368e70c5402cea154cd60bb1272a7f0491fc/templates/logging.json#L29",
```

Comment:
@ameihm0912 could you also add a comment with the location of the mozdef code that uses this new permission? This may currently only exist in a PR but even a link to that would be useful when someone comes back to figure out why this permission was added.